data_dir/longevity_mv_prefilled_profile.yaml

```diff
@@ -0,0 +1,79 @@
+keyspace: mview
```

Comment:
The name of the profile isn't clear.
I don't understand what is prefiiled and what is the diff between this and the other one.

I suggest you call it according to its properties, something like c-s_profile_2_views_7_queries.yaml

Reply:
We need to discuss about the names